action/main.py

```diff
@@ -73,5 +74,6 @@
         stderr=STDOUT,
         encoding="utf-8",
     )
+shutil.rmtree(ENV_PATH)
```

Comment:
Is it worth passing `ignore_errors=True` to this call? An error is unlikely, but it also shouldn't fail the entire action IMO.

Reply:
I agree, I think an error is unlikely but yeah shouldn't fail the entire action.